### ISSUE TITLE
Travis CI: PHPUnit jobs are failing and Travis CI is falsely reporting that jobs pass

### ISSUE THREAD

**Issue by ntwb** [Automattic/jetpack#3955]
Build: https://travis-ci.org/Automattic/jetpack/builds/132665338 Passed "Green" by Travis CI

First PHP 5.6 Job: https://travis-ci.org/Automattic/jetpack/jobs/132377505

The following test failed: https://travis-ci.org/Automattic/jetpack/jobs/132377505#L2662
```shell
Configuration read from /tmp/wordpress/src/wp-content/plugins/jetpack/phpunit.xml.dist
......E........................................................  63 / 421 ( 14%)
............................................................... 126 / 421 ( 29%)
............................................................... 189 / 421 ( 44%)
.........................................................I..... 252 / 421 ( 59%)
..............Warning: No such constant: AUTOMATIC_UPDATER_DISABLED
Warning: No such constant: FS_METHOD
Warning: No such constant: DISALLOW_FILE_EDIT
Warning: No such constant: DISALLOW_FILE_MODS
Warning: No such constant: WP_AUTO_UPDATE_CORE
Warning: No such constant: WP_HTTP_BLOCK_EXTERNAL
Warning: No such constant: WP_ACCESSIBLE_HOSTS
.............S...................SSSS............ 315 / 421 ( 74%)
..............................I................................ 378 / 421 ( 89%)
...............I..I........I.....I.......II
Time: 1.71 minutes, Memory: 83.25Mb
There was 1 error:
1) WP_Test_Jetpack::test_implode_frontend_css_enqueues_bundle_file_handle
filemtime(): stat failed for /tmp/wordpress/src/wp-content/plugins/jetpack/css/jetpack.css
/tmp/wordpress/src/wp-content/plugins/jetpack/class.jetpack.php:6441
/tmp/wordpress/src/wp-content/plugins/jetpack/tests/php/test_class.jetpack.php:176
```

It looks like gulp is not returning the correct error code to Travis CI

**Comment by jeherve** [comment]
cc @kraftbj

**Comment by kraftbj** [comment]
Yeah, I've noticed that too. The constant warnings were due to an incorrect `error_log` called in (proposed fix in #3931) and another case where PHP actually returned a fatal ( #3933 ) but the build passed.

I agree, looks like gulp might be acting up.

**Comment by ntwb** [comment]
#3965 looks good, added a comment in the PR
